helm-charts/orchestrator/templates/network-policies.yaml

```diff
@@ -34,5 +34,23 @@ spec:
           matchLabels:
             # Allow auxiliary knative function for workflow (such as m2k-save-transformation)
             kubernetes.io/metadata.name: knative-serving
-
-{{- end }}
+---
+# NetworkPolicy to allow unrestricted communication between all pods 
+# within the 'sonataflow-infra'
+kind: NetworkPolicy
+metadata:
+  name: allow-intra-sonataflow-infra
```

Comment:
since the namespace might be a different one, let's go with `workflow-intra-namespace`